dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/LicenseStep.java

```diff
@@ -50,15 +51,17 @@ public DataLicense getData(SubmissionService submissionService, InProgressSubmis
     }
 
     @Override
-    public void doPatchProcessing(Context context, Request currentRequest, InProgressSubmission source, Operation op)
-        throws Exception {
+    public void doPatchProcessing(Context context, Request currentRequest, InProgressSubmission source, Operation op,
+                                  SubmissionStepConfig stepConf) throws Exception {
 
         if (op.getPath().endsWith(LICENSE_STEP_OPERATION_ENTRY)) {
 
             PatchOperation<String> patchOperation = new PatchOperationFactory()
                 .instanceOf(LICENSE_STEP_OPERATION_ENTRY, op.getOp());
             patchOperation.perform(context, currentRequest, source, op);
 
+        } else {
+            throw new UnprocessableEntityException("This path : " + op.getPath() + " can not to be replaced");
```

Comment:
If I understand correctly, I think this should likely say `"The path" + op.getPath() + " cannot be patched"`